src/chrome/content/rules/LinkedIn.xml

```diff
@@ -1,4 +1,12 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://ja.linkedin.com/ => https://ja.linkedin.com/: (51, "SSL: no alternative certificate subject name matches target host name 'ja.linkedin.com'")
+Fetch error: http://ko.linkedin.com/ => https://ko.linkedin.com/: (6, 'Could not resolve host: ko.linkedin.com')
+Fetch error: http://th-th.help.linkedin.com/ => https://th-th.help.linkedin.com/: Too many redirects while fetching 'http://th-th.help.linkedin.com/'
+Fetch error: http://asistenta.linkedin.com/ => https://asistenta.linkedin.com/: Too many redirects while fetching 'http://asistenta.linkedin.com/'
+Fetch error: http://marketing.linkedin.com/ => https://marketing.linkedin.com/: Too many redirects while fetching 'https://marketing.linkedin.com/'
+
```

Comment:
ja and ko are mistakes; th-th, asistenta, and marketing redirect back and forth successfully.